lms/static/sass/views/_learner-profile.scss

```diff
@@ -307,8 +308,244 @@
             border: 2px dashed $link-color;
             .u-field-title,
             i {
-              color:  $link-color;
+                color: $link-color;
+            }
+        }
+    }
+
+    .badge-paging-header {
+        padding-top: $baseline;
+    }
+
+    .pagination {
```

Comment:
I thought that there was some pagination code in common, though perhaps the sass is separate. @andy-armstrong, what do you think of this?